spec/models/user_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'spec_helper'
+
+describe User do
+
```

Comment:
Where the tests itself??? When you leave place for future rework please add #ToDo comment